Hermes/HermesNotification.swift

```diff
@@ -19,13 +20,27 @@ public class HermesNotification: NSObject {
       return attributedText?.string
     }
   }
+  
+  /// Text, but with fancy attributes instead of a regular `String`
   public var attributedText: NSAttributedString?
+  
+  /// The color of the bottom bar on the notification
   public var color: UIColor?
+  
+  /// The image to be displayed along with the notification
   public var image: UIImage?
+  
+  /// The URL to the image
```

Comment:
Sorry, I never completed this feature. I should have removed it, but I'll throw in an issue for now.